middleware/multer.js

```diff
@@ -0,0 +1,14 @@
+const multer = require("multer");
+const path = require("path");
+
+module.exports = multer({
+  storage: multer.diskStorage({}),
+  fileFilter: (req, file, cb) => {
+    let ext = path.extname(file.originalname);
+    if (ext.toLowerCase() !== ".jpg" && ext.toLowerCase() !== ".jpeg" && ext.toLowerCase() !== ".png") {
```

Comment:
Consider 
```js
if (![".jpg", ".jpeg", ".png"].includes(ext.toLowerCase()) {
```
More readable and easier to maintain.

Reply:
Also, this would not prevent someone from naming a file `.png` and uploading it, even if it's not an image. I wonder if the library you're using has a method that can perform this check for you?

Reply:
I actually forgot to uninstall multer as I moved my upload request from server side to client side (removed now). I added `resource_type` and `allowed_formats` parameters from Cloudinary library in my main.js for image file validation. Tested manually it seems to work :)